### ISSUE TITLE
Remove support for .NET 6.0

### ISSUE THREAD

**Issue by MichaelSimons** [dotnet/dotnet-docker#6051]
.NET 6.0 reached EOL on Nov 12, 2024.  Per the [Tag Lifecycle documentation](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-tags.md#tag-lifecycle), the 6.0 images will continue to be rebuilt upon base image updates for one month.  After that time, all 6.0 infra, Dockerfiles, tests, documentation, etc. should be removed.  Work can begin in nightly now.

**Comment by mthalman** [comment]
I was _just_ going to log this. 😄

**Comment by tri-bao** [comment]
unfortunately, we still need to stick with .net6 for awhile. Will .net6 docker images still exist in MCR after this ticket is done?

**Comment by richlander** [comment]
Yes. This issues is only about _new_ images. There won't be any. We have no plans to delete any images.

EOL annotations are all you need worry about: https://github.com/dotnet/dotnet-docker/pull/6054.

**Comment by MichaelSimons** [comment]
To re-enforce what @richlander stated.  The following is documented in the [Tag Lifecycle section of the Supported Tags documentation](https://github.com/dotnet/dotnet-docker/blob/main/documentation/supported-tags.md#tag-lifecycle)

> When a .NET version reaches EOL, its tags will continue to be maintained (rebuilt for base image updates) until the next .NET servicing date (typically on "Patch Tuesday", the 2nd Tuesday of the month).
> 
> Once a tag is no longer maintained, it will be considered unsupported, and will no longer be updated. Unsupported tags will continue to exist in the container registry to prevent breaking any references to it.

**Comment by lbussell** [comment]
This is done. Announcement: https://github.com/dotnet/dotnet-docker/discussions/6145
